profiles/views_test.py

```diff
@@ -228,16 +233,22 @@ def test_patch_profile_by_user(client, logged_in_profile):
         "profile_api-detail", kwargs={"user__username": logged_in_profile.user.username}
     )
     # create a dummy image file in memory for upload
+    location_json = {"value": "Boston"}
     with make_temp_image_file(width=250, height=250) as image_file:
         # format patch using multipart upload
         resp = client.patch(
             url,
-            data={"bio": "updated_bio_value", "image_file": image_file},
+            data={
+                "bio": "updated_bio_value",
+                "image_file": image_file,
+                "location": json.dumps(location_json),
```

Comment:
I think you should just be able to pass the nested object in without having to serialize to a string

Reply:
`multipart uploads do not support nested data` (multipart being used here for the simulated image upload).